tests/beacon/test_helpers.py

```diff
@@ -874,24 +893,27 @@ def _create_slashable_vote_data_messages(params):
 def test_verify_slashable_vote_data_signature(privkeys,
                                               sample_beacon_state_params,
                                               genesis_validators,
-                                              sample_slashable_vote_data_params):
+                                              sample_slashable_vote_data_params,
+                                              sample_fork_data_params):
     sample_beacon_state_params["validator_registry"] = genesis_validators
     state = BeaconState(**sample_beacon_state_params)
 
     # NOTE: we can do this before "correcting" the params as they
     # touch disjoint subsets of the provided params
     messages = _create_slashable_vote_data_messages(sample_slashable_vote_data_params)
 
+    fork_data = ForkData(**sample_fork_data_params)
     valid_params = _correct_slashable_vote_data_params(
         sample_slashable_vote_data_params,
         genesis_validators,
         messages,
         privkeys,
+        fork_data,
```

Comment:
The conftest setting `state.fork_data == fork_data = ForkData(**sample_fork_data_params)` is not clear in the tests. I think we should use `state.fork_data` here:
```python
    state = BeaconState(**sample_beacon_state_params).copy(
        validator_registry=genesis_validators,
        fork_data=ForkData(**sample_fork_data_params),
    )

    ...

    valid_params = _correct_slashable_vote_data_params(
        sample_slashable_vote_data_params,
        genesis_validators,
        messages,
        privkeys,
        state.fork_data,
    )
```

Reply:
gotcha